src/types/onyx/Login.ts

```diff
@@ -17,7 +17,9 @@ type Login = OnyxCommon.OfflineFeedback<'defaultLogin' | 'validateLogin' | 'adde
     errorFields?: OnyxCommon.ErrorFields;
 };
 
-type LoginList = Record<string, Login>;
+type LoginWithOfflineFeedback = OnyxCommon.OnyxItemWithOfflineFeedback<Login, keyof Login | 'defaultLogin' | 'validateLogin' | 'addedLogin' | 'deletedLogin'>;
 
-export default Login;
+type LoginList = Record<string, LoginWithOfflineFeedback>;
+
+export default LoginWithOfflineFeedback;
```

Comment:
```suggestion
export default Login;
```

Let's continue to keep using `Login` as it matches with the file name.